app/views/dashboards/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<% content_for :javascript_includes do %>
+  <%= javascript_include_tag "dashboard_poller" %>
```

Comment:
If we change this to be
```
<%= javascript_include_tag "dashboard_poller" unless unless Settings.dsds.dashboard_refresh_disabled %>
```
then we can remove the erb tags from the dashboard_poller file so the javascript is consistent regardless of ENV variables